x-pack/plugins/profiling/common/calculate_impact_estimates/index.ts

```diff
@@ -27,40 +27,52 @@ export function calculateImpactEstimates({
   totalSamples: number;
   totalSeconds: number;
 }) {
-  const annualizedScaleUp = ANNUAL_SECONDS / totalSeconds;
+  return {
+    totalSamples: calculateImpact({
+      samples: totalSamples,
+      totalSamples,
+      totalSeconds,
+    }),
+    totalCPU: calculateImpact({
+      samples: countInclusive,
+      totalSamples,
+      totalSeconds,
+    }),
+    selfCPU: calculateImpact({
+      totalSamples,
+      samples: countExclusive,
```

Comment:
```suggestion
      samples: countExclusive,
      totalSamples,
```

Could we swap these lines? On first reading, it took me a second to decode this for some reason since the expected order was different than the previous two.